StyleCop.Analyzers/StyleCop.Analyzers/Helpers/TaskEx.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace StyleCop.Analyzers.Helpers
+{
+    using System.Threading.Tasks;
+
+    internal static class TaskEx
+    {
+        internal static Task CompletedTask { get; } = Task.FromResult(true);
```

Comment:
:bulb: Rather than return `Task.FromResult(true)`, return `Task.FromResult(default(VoidResult));`, by including [this `VoidResult`](https://github.com/rackerlabs/dotnet-threading/blob/master/Rackspace.Threading/VoidResult.cs) as a nested class.